aiida_orca/parsers/__init__.py

```diff
@@ -70,6 +70,15 @@ def _remove_nan(parsed_dictionary: dict) -> dict:
                     non_nan_value = np.nan_to_num(value, nan=123456789, posinf=2e308, neginf=-2e308)
                     parsed_dictionary.update({key: non_nan_value})
 
+            # Hack for full TDDFT calculations without the TDA approximation,
+            # which do not provide CI coefficients and cclib parser returns NaNs.
+            # In this case we're deleting the entry,
+            # which seems safer than returning bogus info.
+            # This is not handled by the code above
+            # because the value is not a numpy array.
+            if 'etsecs' in parsed_dictionary and np.isnan(parsed_dictionary['etsecs'][0][0][-1]):
+                del parsed_dictionary['etsecs']
+
```

Comment:
I think this needs addition of metadata to let the user know about this hack. This can go into `warning`.

Reply:
Hmm, I'd rather keep the `warning` field for actual warnings coming from ORCA that are actionable from the user (e.g. one can get rid of them by modififying the input file). In this case, ORCA simply does not provide this info, and since we are deleting the key, the user is sure to notice. I've added a logging statement though, LMK what you think.